### ISSUE TITLE
[cmake] misleading indentation in hash_map.h (Trac #1659)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1651]
```text
/home/dschultz/Documents/combo/trunk/src/cmake/tool-patches/common/I3/hash_map.h:13:5: warning: this ‘while’ clause does not guard... [-Wmisleading-indentation]
     while ((c = *str++))
     ^~~~~
/home/dschultz/Documents/combo/trunk/src/cmake/tool-patches/common/I3/hash_map.h:15:7: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘while’
       return hash;
       ^~~~~~
```

Migrated from https://code.icecube.wisc.edu/ticket/1659
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "{{{\n/home/dschultz/Documents/combo/trunk/src/cmake/tool-patches/common/I3/hash_map.h:13:5: warning: this \u2018while\u2019 clause does not guard... [-Wmisleading-indentation]\n     while ((c = *str++))\n     ^~~~~\n/home/dschultz/Documents/combo/trunk/src/cmake/tool-patches/common/I3/hash_map.h:15:7: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018while\u2019\n       return hash;\n       ^~~~~~\n}}}", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "cmake", 
    "summary": "[cmake] misleading indentation in hash_map.h", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2016-04-26T19:54:40", 
    "milestone": "", 
    "owner": "nega", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160426T20:02:20: 
* cweaver changed resolution from "" to "fixed"
* cweaver commented:

> In changeset 2529:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2529"
> Remove extra whitespace to reassure the compiler
> Closes #1651
> ```


* cweaver changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:15:43: nwhitehorn commented:

> In changeset 170251:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170251"
> Remove extra whitespace to reassure the compiler
> Closes #1651
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas changed milestone from "June 2016" to ""
* olivas commented:

> Milestone June 2016 deleted
